source/manual/cdn.html.md

```diff
@@ -28,55 +25,37 @@ Most of the CDN config is versioned and scripted:
 - [CDN configuration](https://github.com/alphagov/govuk-cdn-config/)
 - [CDN config secrets](https://github.com/alphagov/govuk-cdn-config-secrets)
 
-These are deployed to [integration][integration_cdn], [staging][staging_cdn]
-and [production][production_cdn].
+These are deployed to [integration][integration_cdn], [staging][staging_cdn] and [production][production_cdn].
 
-Some configuration isn't scripted, such as logging. The www, bouncer and assets
-services sends logs to S3 and stream them to `monitoring-1`. These logging
-endpoints are configured directly in the Fastly UI. There is
-[documentation](/manual/query-cdn-logs.html) on how to query the CDN logs.
+Some configuration isn't scripted, such as logging. The www, bouncer and assets services sends logs to S3 and stream them to `monitoring-1`. These logging endpoints are configured directly in the Fastly UI. There is [documentation](/manual/query-cdn-logs.html) on how to query the CDN logs.
 
 [integration_cdn]: https://deploy.integration.publishing.service.gov.uk/job/Deploy_CDN/
 [staging_cdn]: https://deploy.staging.publishing.service.gov.uk/job/Deploy_CDN/
 [production_cdn]: https://deploy.publishing.service.gov.uk/job/Deploy_CDN/
 
 ## Fastly Caching
 
-The main www.gov.uk cache is
-[Varnish](https://varnish-cache.org/docs/2.1/index.html), which Fastly run for
-us.
+The main www.gov.uk cache is [Varnish](https://varnish-cache.org/docs/2.1/index.html), which Fastly run for us.
 
 Varnish lets us configure our caching logic with VCL (Varnish config language).
 
-It also lets us do fancy things, like [only allowing connections to staging from permitted IPs](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L193), [forcing SSL](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L214) and [blocking IP addresses](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L200), among other things.
+It also lets us do fancy things, like [only allowing connections to staging from permitted IPs](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L202), [forcing SSL](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L222) and [blocking IP addresses](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L208), among other things.
```

Comment:
Is it worth linking to a specific commit hash rather than `master`, as this is likely to change again in future?

Reply:
Good idea! I've pointed it at the latest commit, since these are cited as examples and it doesn't matter if they go a bit out of date.